spec/call_spec.rb

```diff
@@ -39,7 +39,7 @@
     expect(voice_client)
       .to receive(:request)
       .with(:post, 'calls', { source: source, destination: destination, callFlow: call_flow.to_json })
-      .and_return('{"data":[{"id":"' + call_id + '","status":"queued","source":"' + source + '","destination":"' + destination + '","createdAt":"2019-10-11T13:02:19Z","updatedAt":"2019-10-11T13:02:19Z","endedAt":null}],"_links":{"self":"/calls/' + call_id + '"},"pagination":{"total_count":0,"pageCount":0,"currentPage":0,"perPage":0}}')
+      .and_return("{\"data\":[{\"id\":\"#{call_id}\",\"status\":\"queued\",\"source\":\"#{source}\",\"destination\":\"#{destination}\",\"createdAt\":\"2019-10-11T13:02:19Z\",\"updatedAt\":\"2019-10-11T13:02:19Z\",\"endedAt\":null}],\"_links\":{\"self\":\"/calls/#{call_id}\"},\"pagination\":{\"total_count\":0,\"pageCount\":0,\"currentPage\":0,\"perPage\":0}}")
```

Comment:
Does it work with single quotes for the internal quotes?

```suggestion
      .and_return("{'data':[{'id':'#{call_id}','status':'queued','source':'#{source}','destination':'#{destination}','createdAt':'2019-10-11T13:02:19Z','updatedAt':'2019-10-11T13:02:19Z','endedAt':null}],'_links':{'self':'/calls/#{call_id}'},'pagination':{'total_count':0,'pageCount':0,'currentPage':0,'perPage':0}}")
```